omod/src/main/java/org/openmrs/module/reportingrest/web/wrapper/RunReportRequest.java

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
This class should not be needed.  It doesn't have anything that the existing ReportRequest object representation doesn't already have.

Reply:
I left this class and changed name to RunReportRequest**DTO** to emphasize the purpose of this class. 
From frontend we send just few simple values so I think it will be more efficient to use DTO class instead of ReportRequest object that contains a lot of fields.
Moreover I noticed issues when we try to use ReportRequest object as an endpoint body when we don't fill this object fully (e.g. RenderingMode object throws NPE in toString() method because ReportRenderer field is used in this method but this field will never be filled by the frontend) so simple DTO with necessary fields will be more efficient and less susceptible to possible errors

Reply:
Probably my bad, changed to use existing ReportRequest class. This new RunReportRequest class has been removed.